src/entry.c

```diff
@@ -35,9 +35,13 @@ void free_entry(struct entry_t *entry)
     entry->vtable->free(entry);
 }
 
-int is_entry_selectionable(struct entry_t *entry)
+int is_entry_selectable(struct entry_t *entry)
 {
-    return entry->vtable->is_selectionable(entry);
+    return entry->vtable->is_selectable(entry);
 }
 
+void *get_type(struct entry_t *entry, entry_type_t type)
+{
+    return entry->vtable->get_type(entry, type);
+}
 
```

Comment:
I'm not sure about this get_type() which seems to be redundant with is_selectable()

Reply:
It's not the same, because I changed the `is_selectable()` implementation in line.c. For example lines generate by the -A, -B or -C arguments are not selectable and cannot be opened.